src/main/proto/admin.proto

```diff
@@ -66,6 +67,7 @@ message ContextOverview {
   map<string, string> meta_data = 3;
   int64 pendingSince = 4;
   bool changePending = 5;
+  bool dcbContext = 6;
```

Comment:
Does this mean you can update a context to become a DCB Context?

Reply:
@MGathier ?